src/tools/sdk/index.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Dart SDK
+description: Dart libraries and command-line tools.
+permalink: /tools/sdk
+js:
+- url: /tools/sdk/archive/assets/install.js
+  defer: true
+---
+
+The Dart SDK has the libraries and command-line tools that you need to develop
+Dart apps. To learn about other tools you can use for Dart development, see
+[Dart Tools](/tools).
+
+The [DartPad][]{: target="_blank"} is a great way to learn how to write a simple
```

Comment:
This paragraph (or maybe something more like the original)  belongs on the main Install page, not here.